Macros/ACS_summary_geo_source.sas

```diff
@@ -2657,6 +2716,23 @@
     
     NumRenterHsgUnits_&_years. = NumRenterOccupiedHU_&_years. + NumVacantHsgUnitsForRent_&_years.;
 
+	NumRenterCostBurden_&_years. = sum(B25070e7, B25070e8, B25070e9, B25070e10)
+	NumRentSevereCostBurden_&_years. = B25070e10
+	NumOwnerCostBurden_&_years. = sum(B25091e8, B25091e9, B25091e10, B25091e11, B25091e19, B25091e20, B25091e21, B25091e22)
+	NumOwnSevereCostBurden_&_years. = sum(B25091e11, B25091e22)
+
```

Comment:
We need to add vars for the denominators for these cost burden calculations. The denominator is all renter and all owner units, respectively, excluding the "Not computed" cells.